### PR TITLE
feat: language settings #586

### PR DESCRIPTION
**Pull request by FrolYo**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Chat.UI.Blazor.csproj [hunk 1]
```diff
@@ -24,24 +24,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="Components\ChatSettings\**" />
+    <_ContentIncludedByDefault Remove="Components\Radio\Radio.razor" />
```
Comment (alexis-kochetov) [inline src/dotnet/Chat.UI.Blazor/Chat.UI.Blazor.csproj:27]: Why do we need `Remove` at all?

File: src/dotnet/Core/Identifiers/LanguageId.cs [hunk 2]
```diff
@@ -3,30 +3,56 @@ namespace ActualChat;
...
             "en-US" => "EN",
+            "fr-FR" => "FR",
+            "de-DE" => "DE",
             "ru-RU" => "RU",
+            "es-ES" => "ES",
+            "uk-UA" => "UK",
             _ => "?",
         };
 
+    // TODO: use Settings instead
```
Comment (alexis-kochetov) [inline src/dotnet/Core/Identifiers/LanguageId.cs:55]: Let's remove

**Review by alexis-kochetov — APPROVED** [review]

File: src/dotnet/Core/Identifiers/LanguageId.cs (hunk 2 above)
Comment (FrolYo) [reply src/dotnet/Core/Identifiers/LanguageId.cs:55]: done

File: src/dotnet/Chat.UI.Blazor/Chat.UI.Blazor.csproj (hunk 1 above)
Comment (FrolYo) [reply src/dotnet/Chat.UI.Blazor/Chat.UI.Blazor.csproj]: done

**Pull request merged**